example/controller-registration.yaml

```diff
@@ -1,11 +1,10 @@
 ---
-apiVersion: core.gardener.cloud/v1beta1
+apiVersion: core.gardener.cloud/v1
```

Comment:
@Nuckal777 In your current setup can you use the `v1` version of the `ControllerDeployment`?

Reply:
Yes, can be merged.